pkgs/development/tools/github/cligh/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, pythonPackages }:
+
+with pythonPackages;
+buildPythonPackage rec {
+  name = "cligh-${version}";
+  version = "0.3";
+  
+  namePrefix = "";
+ 
+  src = fetchurl {
+    url = "https://github.com/CMB/cligh/archive/v${version}.tar.gz";
```

Comment:
OK. I changed it to fetchFromGiHub. I thought the release archive versions may have different contents so fetchurl was used, but now I see their release archives seem to have the same content as in the git repos.